packages/mds-api-authorizer/index.ts

```diff
@@ -14,11 +14,41 @@ const {
   TOKEN_USER_EMAIL_CLAIM = 'https://ladot.io/user_email'
 } = process.env
 
-export type ApiAuthorizer = (req: express.Request) => ApiAuthorizerClaims | null
+export type ApiAuthorizer = (req: express.Request | string) => ApiAuthorizerClaims | null
 
 export const AuthorizationHeaderApiAuthorizer: ApiAuthorizer = req => {
-  if (req.headers && req.headers.authorization) {
-    const [scheme, token] = req.headers.authorization.split(' ')
+  if (typeof req !== 'string') {
+    if (req.headers && req.headers.authorization) {
+      const [scheme, token] = req.headers.authorization.split(' ')
+      const decoders: { [scheme: string]: () => ApiAuthorizerClaims } = {
```

Comment:
We could probably pull this logic out into a helper method like `getDecodersForSchemes()`